pkg/minikube/cluster/cluster.go

```diff
@@ -206,9 +207,23 @@ func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 		driver = createVirtualboxHost(config)
 	case "vmwarefusion":
 		driver = createVMwareFusionHost(config)
-	case "kvm", "kvm2":
+	case "kvm":
+		if viper.GetBool(cfg.ShowDriverDeprecationNotification) {
+			fmt.Fprintln(os.Stderr, `WARNING: The kvm driver is now deprecated and support for it will be removed in a future release.
+				Please consider switching to the kvm2 driver, which is intended to replace the kvm driver.
+				See https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#kvm2-driver for more information.
```

Comment:
I do not see a topic in the docs related to `kvm2`

Reply:
Nice catch. We'll get that in today too.